### PR TITLE
feat: Add measurement widget (in progress)

### PR DESCRIPTION
**Pull request by kellyhutchins**
In Progress: 

Todo: 

- Add support for coordinate formats 
- Troubleshoot why esri css isn't applied to measure widget
- Add css variables (if needed) to style components

### CODE REVIEW

**Comment by kellyhutchins** [comment]
Added measurement widget 
Broke test apps out into individual files. They should probably go into a demos/samples folder but was having issues with the assets.

File: src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
+/** @license
+ * Copyright 2022 Esri
```
Comment (rslibed) [inline src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx:2]: Update to 2023

File: src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+const CSS = {
+  base,
+  hide: `${base}__hide`,
+};
+@Component({
+  tag: 'instant-apps-measurement-tool',
+  styleUrl: 'instant-apps-measurement-tool.scss',
+  scoped: true,
+})
+export class MeasurementTool {
```
Comment (rslibed) [inline src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx:31]: Update to InstantAppsMeasurementTool for consistent naming across the components

**Review by rslibed — APPROVED** [review]
Looks good! Left some comments. 

We should also increment the version number to `1.0.0-beta.58` in the package.json:
https://github.com/Esri/instant-apps-components/blob/master/package.json

And summarize updates in the CHANGELOG.md:
https://github.com/Esri/instant-apps-components/blob/master/CHANGELOG.md

File: src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx (hunk 1 above)
Comment (kellyhutchins) [reply src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx]: Good catch. Updated.

**Pull request merged**